components/more-less/test/more-less.visual-diff.js

```diff
@@ -7,6 +7,25 @@ describe('d2l-more-less', () => {
 
 	let browser, page;
 
+	const contentResize = (page, selector) => {
+		return page.evaluate((selector) => {
+			return new Promise((resolve) => {
+				const elem = document.querySelector(selector);
+				elem.shadowRoot.querySelector('.d2l-more-less-content').addEventListener('transitionend', (e) => {
+					if (e.propertyName === 'height') resolve();
```

Comment:
This one could almost be simplified like the other one, except we'd have to enhance the `oneEvent` helper to accept some type of option/predicate to check the property name.  Typically this check is made when there could be multiple events being triggered.  If we did enhance that function, we'd have to be careful though - it can't be a function predicate that is passed... whatever is passed into `evaluate`/`$eval`/`$$eval` mist be serializable. Again, technically beyond your current goal here.  :)